### DISCUSSION TITLE
percy cli snapshot error

### DISCUSSION THREAD

**Discussion by geryit** [percy/cli#784] [Support] [answered]
## The problem

We get this error when we run `percy snapshot`
```
yarn percy snapshot https://percy.io/sitemap.xml
  yarn run v1.22.17
$ /node_modules/.bin/percy snapshot https://percy.io/sitemap.xml
  [percy] Error: Cannot find module '/node_modules/@percy/agent/dist/commands/exec.d'
Require stack:
  - /node_modules/@percy/cli/dist/commands.js
- /node_modules/@percy/cli/dist/percy.js
- /node_modules/@percy/cli/dist/index.js
- /node_modules/@percy/cli/bin/run
```
## Environment

- Node version: `v16.14.0`
- `@percy/cli` version: `^1.0.0-beta.75`
- Version of Percy SDK you’re using: `cli`
- If needed, a build or snapshot ID:
- OS version: `macOS`
- Type of shell command-line [interface]: `shell`

**Comment by Robdel12** [comment, marked as answer]
Hey @geryit! Looking at the stack trace, there’s `@percy/agent` still around in your project or env. Percy CLI completely replaces agent so I’d make sure agent is removed from everywhere

**Reply by geryit** [reply]
@Robdel12 I dont have `@percy/agent` in `package.json`. Is there another package loading this?

**Reply by geryit** [reply]
I had `"@percy/script": "^1.1.0",` in packages. removed it and it works now
